suzieq/poller/coalescer.py

```diff
@@ -0,0 +1,93 @@
+import errno
```

Comment:
I don't quite like the name coalescer.py because it gives the wrong impression that this is the real coalescer. Its small enough to put it in the poller?  Not completely against it, i.e. you can convince me to pursue your approach, but I need a little convincing.

Should we move sq_coalescer.py to this directory?